packages/shared/web/src/dashboard-view/hooks/use-liquidity-leveraged-grid.tsx

```diff
@@ -66,12 +110,29 @@ export const useLiquidityLeveragedGrid = (
             ),
         incentiveSymbols: currentPosition
           ? getTotalIncentiveSymbol(
-              currentPosition?.asset.marketYield?.secondaryIncentives?.symbol,
-              currentPosition?.asset.marketYield?.noteIncentives?.symbol
+              currentPosition?.asset.marketYield?.secondaryIncentives
```

Comment:
There is no difference between the `marketYield` object and the `y` object. You don't need to do a `?` on the currentPosition state here. You can just use `getTotalIncentiveSymbol(y?.....)` for everything.

Same for the if clause above.